lib/rubocop/result_cache.rb

```diff
@@ -188,14 +194,24 @@ def rubocop_checksum
             .select { |path| File.file?(path) }
             .sort!
             .each do |path|
-              content = File.binread(path)
-              digest << Zlib.crc32(content).to_s # mtime not reliable
+              digest << digest(path)
             end
           digest << RuboCop::Version::STRING << RuboCop::AST::Version::STRING
           digest.hexdigest
         end
     end
 
+    def digest(path)
+      if path.end_with?(*DL_EXTENSIONS)
+        # shared libraries often contain timestamps of when
```

Comment:
I'd like this to be a sentence, so `Shared` instead of `shared` and a dot at the end.